src/openzeppelin/token/erc20.cairo

```diff
@@ -1,23 +1,36 @@
 use starknet::ContractAddress;
 
+#[abi]
 trait IERC20 {
+    #[view]
     fn name() -> felt252;
+    #[view]
     fn symbol() -> felt252;
+    #[view]
     fn decimals() -> u8;
+    #[view]
     fn total_supply() -> u256;
+    #[view]
     fn balance_of(account: ContractAddress) -> u256;
+    #[view]
     fn allowance(owner: ContractAddress, spender: ContractAddress) -> u256;
+    #[external]
     fn transfer(recipient: ContractAddress, amount: u256) -> bool;
+    #[external]
     fn transfer_from(sender: ContractAddress, recipient: ContractAddress, amount: u256) -> bool;
+    #[external]
     fn approve(spender: ContractAddress, amount: u256) -> bool;
+    #[external]
+    fn increase_allowance(spender: ContractAddress, added_value: u256) -> bool;
+    #[external]
+    fn decrease_allowance(spender: ContractAddress, subtracted_value: u256) -> bool;
```

Comment:
Same issue with the Account interface, I think this is a naming issue, the `trait IERC20` IMO should not be describing the interface of the ERC, but the interface of our implementation, because developers need that interface for Dispatcher calls. I understand the confusion this may generate, maybe we should call these abi traits something like OZ_ERC20 instead? What are your thoughts on this @martriay

Reply:
If the pattern I proposed for the Account interface is accepted I will update this accordingly